circle.yml

```diff
@@ -119,7 +119,7 @@ commands:
       - run:
           name: Run unit-test, generate Jacoco test report and Post code coverage report to Codecov.io
           command: |
-            ./gradlew testDebugUnitTestCoverage
+            ./gradlew jacocoTestReleaseUnitTestReport
```

Comment:
I opted to use release build for codecov base instead debug, any issues this might cause?

Reply:
Yeah 👀 https://github.com/codecov/example-android#instrumentation-tests-coverage-reports we should enable `testCoverageEnabled true` for the `release` `buildType` in https://github.com/mapbox/mapbox-navigation-android/blob/fad1cd4de295399b999dc5009b66cad2e567b01a/libandroid-navigation/build.gradle#L26-L30 The only thing that I'm unsure about is if Codecov will retrieve the reports seamlessly or if we need to set the release path explicitly, although I imagine that it'll 👀 to `build/reports/jacoco` and get all the reports for the different variants so I believe we should be good.

Reply:
Reverted back to debug and used the flag in only one place.